### DISCUSSION TITLE
Release candidates of Saleor 2.11

### DISCUSSION THREAD

**Discussion by karolkielecki** [saleor/saleor#5999] [Show and tell]
We've rolled out the first release candidates of Saleor 2.11: [Dashboard](https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.0-rc.1) and [Core API](https://github.com/mirumee/saleor/releases/tag/2.11.0-rc.1). Storefront 2.11 RC.1 will be available next week. If you have any feedback, or issues, feel free to let us know!

**Comment by karolkielecki** [comment]
The new release candidates for Saleor 2.11 are out! Have a look and let us know if you have any feedback.

- https://github.com/mirumee/saleor/releases/tag/2.11.0-rc.3
- https://github.com/mirumee/saleor-storefront/releases/tag/2.11.0-rc.1
- https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.0-rc.3

**Comment by patrys** [comment]
Most recent versions:

* https://github.com/mirumee/saleor/releases/tag/2.11.0-rc.5
* https://github.com/mirumee/saleor-storefront/releases/tag/2.11.0-rc.3
* https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.0-rc.5

**Comment by karolkielecki** [comment]
The latest release candidates of Saleor 2.11: 

- https://github.com/mirumee/saleor/releases/tag/2.11.0-rc.6
- https://github.com/mirumee/saleor-storefront/releases/tag/2.11.0-rc.4
- https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.0-rc.6

**Comment by karolkielecki** [comment]
The latest release candidates of Saleor 2.11:

- https://github.com/mirumee/saleor/releases/tag/2.11.0-rc.7
- https://github.com/mirumee/saleor-storefront/releases/tag/2.11.0-rc.5
- https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.0-rc.7

**Comment by karolkielecki** [comment]
We've rolled out new release candidates for 2.11. Learn more about what's new in the release notes. If you have any feedback, feel free to let us know.

- https://github.com/mirumee/saleor/releases/tag/2.11.0-rc.8

- https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.0-rc.8

**Comment by karolkielecki** [comment]
New release candidates of Saleor 2.11 are available for testing. All feedback is appreciated!

- Core API: https://github.com/mirumee/saleor/releases/tag/2.11.0-rc.9
- Dashboard: https://github.com/mirumee/saleor-dashboard/releases/tag/2.11.0-rc.9
